package.json

```
@@ -48,8 +50,8 @@
     "coverage:ci": "node packages/build/bin/run-nyc report --reporter=text-lcov | coveralls",
     "precoverage": "npm test",
     "coverage": "open-cli coverage/index.html",
-    "lint": "npm run prettier:check && npm run eslint && node bin/check-package-locks",
-    "lint:fix": "npm run eslint:fix && npm run prettier:fix",
+    "lint": "npm run eslint && node bin/check-package-locks",
+    "lint:fix": "npm run eslint:fix",
     "eslint": "node packages/build/bin/run-eslint --report-unused-disable-directives --cache .",
     "eslint:fix": "npm run eslint -- --fix",
     "prettier:cli": "node packages/build/bin/run-prettier \"**/*.ts\" \"**/*.js\" \"**/*.md\"",
```

I realized just today that by moving all Prettier linting to eslint, we are no longer going to check markdown files (`**/*.md`).

I am proposing to keep `prettier` as part of our linter checks, but change `prettier:cli` to scan only markdown files.

```suggestion
    "prettier:cli": "node packages/build/bin/run-prettier \"**/*.md\"",
```

---

IMO, we should keep the npm scripts to run full prettier check/fix.

---

So, shall we leave as-is for "prettier:cli"?

---

Let's discuss the high-level user experience we are aiming for.

Personally, I'd like the following behavior:

- When I run `npm test` or `npm run lint`, pretty formatting of JavaScript & TypeScript files is handled by `eslint` to leverage the caching mechanism and Markdown files are checked by `prettier`.

In other words, I'd like to use caching mechanism to skip checking files that are known to be valid. Prettier does not support caching yet, but eslint does, that's why I'd like to check pretty formatting using eslint. However, eslint cannot check Markdown files, we have to keep using `prettier` for this step.